app/views/partials/_navbar.html.erb

```diff
@@ -2,11 +2,12 @@
             <div class="name">Recipe App</div>
             <% if user_signed_in? %>
             <ul class="list">
-                <li class="item"><a class="nav-link"><%= link_to "Food", user_foods_path(current_user), method: :get%></a></li>
-                <li class="item"><a class="nav-link"><%= link_to "Recipies", user_recipes_path(current_user), method: :get%></a></li>
-                <li class="item"><a class="nav-link" href="#portfolio-form-link"><%= link_to "Public Recipies", public_recipes_path, method: :get%></a></li>
-                <li class="item"><a class="nav-link" href="#portfolio-form-link"><%= link_to "Shopping List", user_shopping_lists_path(current_user), method: :get%></a></li>
-                <li class="item"><a class="nav-link" href="#portfolio-form-link"><%= link_to "Log Out", destroy_user_session_path%></a></li>
+                <li class="item"><a class="nav-link"><%= link_to "Food", user_foods_path, method: :get, class: 'link'%></a></li>
+                <li class="item"><a class="nav-link"><%= link_to "Recipies", user_recipes_path, method: :get, class: 'link'%></a></li>
+                <li class="item"><a class="nav-link" href="#portfolio-form-link"><%= link_to "Public Recipies", user_public_recipes_path, method: :get, class: 'link'%></a></li>
+                <li class="item"><a class="nav-link" href="#portfolio-form-link"><%= link_to "Shopping List", user_shopping_lists_path, method: :get, class: 'link'%></a></li>
+                <li class="item"><a class="nav-link" href="#portfolio-form-link"><%= link_to "Log Out", destroy_user_session_path, class: 'link'%></a></li>
```

Comment:
In some links is asking for the user for the routes, I recommend adding `current_user` to the routes

![image](https://user-images.githubusercontent.com/93499010/181387583-3e0f2e4f-9c4a-47d3-8f48-f0d4572d36c6.png)